.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,123 @@
+name: 🐛 Bug Report
+description: Report a bug that affects Lit or a Lit library/tool
+
+body:
+  - type: dropdown
+    id: affected
+    validations:
+      required: true
+    attributes:
+      label: Which package(s) are affected?
+      multiple: true
+      description: Note you can select more than one.
+      options:
+        - Lit Core (lit/lit-html/reactive-element/lit-element)
+        - Context (@lit-labs/context)
+        - Eleventy Plugin (@lit-labs/eleventy-plugin-lit)
+        - Localize (@lit/localize)
+        - Motion (@lit-labs/motion)
+        - Observers (@lit-labs/observers)
+        - React (@lit-labs/react)
+        - SSR (@lit-labs/ssr)
+        - Starter Kits (@lit/lit-starter-(js|ts))
+        - Task (@lit-labs/task)
+        - Other/unknown
+
+  - type: textarea
+    id: description
+    validations:
+      required: true
+    attributes:
+      label: Description
+      description: |
+        Please describe the bug in as much detail as possible. Include what you expected to happen, and what actually happened. If you have an error message, please include as much of the error message as possible.
+      placeholder: |
+        When I register an event listener on a button like this:
+
+        ```html
+        <button @click=${this._onClickButton}>Click me</button>
+        ```
+
+        I would expect the `_onClickButton` event handler to fire when the button is clicked.
+
+        However, clicking on the button doesn't fire the event handler, and I see the following error in the console:
+
+        ```
+        repro.js:23 Uncaught TypeError: Cannot read properties of undefined (reading 'numClicks')
+        at XRepro._onClickButton (repro.js:23:24)
+        at EventPart.handleEvent (lit-html.js:1383:35)
+        ```
+
+  - type: textarea
+    id: reproduction
+    validations:
+      required: true
+    attributes:
+      label: Reproduction
+      description: |
+        Please link to a working reproduction that will help us verify and understand the bug.
+
+        For bugs that show up in the browser, please provide a link to the lit.dev playground: [TypeScript example](https://lit.dev/playground/#gist=b90fc1b4d2ee57120763d2e7db934511), [JavaScript example](https://lit.dev/playground/#gist=2d20e91832d3f98332bf23ced55bea04). Click the "Share" button to generate a URL.
```

Comment:
I wonder whether we want to say anything along the lines of "If you are unable to create a reproduction in the lit.dev playground but can provide a link to a working reproduction hosted elsewhere, please do so."

On the one hand, I really want to encourage people to use the playground, but on the other, any working repro is much better than none.

Reply:
Done